inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/SinkInfoUtils.java

```diff
@@ -182,13 +183,15 @@ private static HiveSinkInfo createHiveSinkInfo(HiveSinkResponse hiveInfo, List<F
             }).collect(Collectors.toList());
         }
 
-        // dataPath = dataPath + / + tableName
-        StringBuilder dataPathBuilder = new StringBuilder();
+        // dataPath = dataPath + / + dbName + / + tableName
```

Comment:
We use the data path and table name as the real path to writing files.
If changed, maybe occurred an exception in other cases.

Reply:
Maybe you can change the usage of the dataPath when creating Hive tables.

Reply:
Added compatibility for the default db case.

No matter where  you create hive table the table name must be part of the path and must also be specified here to let sort write to the correct file. The default hive approach is fine. All that's left is to take the default database case into account.